util/sqlutil/main.go

```diff
@@ -59,9 +62,8 @@ func (t *T) columnLike(s string) string {
 }
 
 func marshal(v interface{}) (interface{}, error) {
-	m, isM := v.(db.Marshaler)
 
-	if isM {
+	if m, isMarshaler := v.(db.Marshaler); isMarshaler {
```

Comment:
I think `if m, ok := v.(db.Marshaler); ok {` is more common, but that just stylistic ..